src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -109,6 +114,18 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
     } else {
       if (verificationStatus === "VERIFIED_WITH_CHANGES") {
         setModalType(ModalType.SUGGESTIONS)
+
+        trackEvent({
+          action_type: "validationAddressViewed",
+          context_module: "OrdersShipping",
+          context_page_owner_type: "orders-shipping",
+          context_page_owner_id: "57e60c68-a198-431e-8a02-6ecb01e3a99b",
```

Comment:
**suggestion:** I'm not too excited about having to pass an order ID to this component just to populate this event. Perhaps we can fire this event on the [`onClose` and `onChosenAddress` props](https://github.com/artsy/force/blob/ce6b42bdef12bc07be7d432ec89507324a73a63e/src/Apps/Order/Routes/Shipping/index.tsx#L769-L777) that are passed in from the parent component?

Reply:
hey, so I totally understand not wanting to pass the ids as props to the AddressVerificationFlow, but looking through the flow there are some distinctions that need to be made for each tracking event. We want to be able to log an event for each modalType based on user action (whether they see the review or suggestions modals) so that's why I included them here.

Reply:
I was able to useSystemContext() and useAnalyticsContext() to get access to the user_id & ownerrType so I believe we're all good here!

Reply:
Figured this out with some help from @rquartararo- the `contextPageOwnerSlug` coming from systemContext() provides the order-id and that's what I'll use here.